### PR TITLE
Deploy Clear to DigitalOcean guide.

### PR DESCRIPTION
**Pull request by bktan8**
Signed-off-by: Bun K Tan <bun.k.tan@intel.com>

### CODE REVIEW

**Comment by mvincerx** [comment]
@bktan8 I see you squashed commits. Is this now ready for review?

**Comment by bktan8** [comment]
yes, pls!

**Comment by mvincerx** [comment]
OK. Will review today. Juggling like usual. 🤹‍♂

File: source/get-started/cloud-install/digitalocean.rst [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+   The output should be :file:`clear-digitalocean.img.gz`.
+
+   .. note::
+      
+      *bzip2* is the other compression format DigitalOcean accepts.
+
+Upload image
+============
+
+#. On DigitalOcean's website, go to :guilabel:`MANAGE` > :guilabel:`Images`
```
Comment (mvincerx) [inline source/get-started/cloud-install/digitalocean.rst]: Use `:menuselection:`MANAGE > Images`` here.

File: source/get-started/cloud-install/digitalocean.rst [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+#. Get the IP address of your |CL| instance.  
+
+   .. figure:: ../../_figures/digitalocean/11-digitalocean.png
+      :scale: 100 %
+      :alt: DigitalOcean - Get Droplet IP address
+
+      Figure 11: DigitalOcean - Get Droplet IP address
+
+#. On your client system, SSH into your instance.
+
```
Comment (mvincerx) [inline source/get-started/cloud-install/digitalocean.rst]: Can you share how to access your password from within DO?

**Comment by mvincerx** [comment]
Thanks for all the work on this new guide. LGTM.

**Pull request merged**